### PR TITLE
Add support for setting cover with MPRIS

### PR DESCRIPTION
**Pull request by ilya-fedin**

### CODE REVIEW

File: Telegram/SourceFiles/platform/linux/linux_mpris_support.cpp [hunk 1]
```diff
@@ -119,6 +127,25 @@ auto CreateMetadata(const Media::Player::TrackState &state) {
...
+				QByteArray thumbnailData;
+				QBuffer thumbnailBuffer(&thumbnailData);
+				trackView->thumbnail()->original().save(
+					&thumbnailBuffer,
+					"JPG",
+					87);
+
+				result["mpris:artUrl"] = Glib::Variant<
+					Glib::ustring
+				>::create("data:image/jpeg;base64,"
```
Comment (JuniorJPDJ) [inline Telegram/SourceFiles/platform/linux/linux_mpris_support.cpp:143]: Are you sure you can just set this mimetype to jpeg?

Comment (ilya-fedin) [reply Telegram/SourceFiles/platform/linux/linux_mpris_support.cpp:143]: look multiple lines above...

Comment (JuniorJPDJ) [reply Telegram/SourceFiles/platform/linux/linux_mpris_support.cpp]: oh sorry, you are right, I missed this save format.

**Comment by john-preston** [comment]
Thanks!

**Pull request merged**